transport_nantes/mobilito/models.py

```diff
@@ -111,3 +111,18 @@ def save(self, *args, **kwargs) -> None:
             super().save(*args, **kwargs)
         except ValidationError as e:
             logger.error(f"Error while saving event : {e}")
+
+
+class SessionReport(models.Model):
+    """A report on a session."""
```

Comment:
This comment doesn't seem to add much value over the class name?

Reply:
# This is a comment 
sort of vibe haha, added it just to have a docstring, but that wasn't exactly necessary.

Reply:
Or
```
Represent user spam/abuse reports on mobilito recording sessions.

We are publishing user-provided data to the internet.
Despite our efforts, it may happen that some sessions
are inappropriate, off brand, and/or shouldn't appear
publicly.  This mechanism lets visitors flag such inappropriate sessions with an optional note to tell us
why they believe the session is inappropriate.
```